terraform/modules/monitoring/grafana/inputs.tf

```diff
@@ -0,0 +1,14 @@
+variable "image" {
+  type = string
+}
+
+variable "network" {
+  type = string
+
```

Comment:
I don't remember if we should put defaut values here ! Can you look in the other modules and do the same ?

Reply:
Ok for me